ipwb/__main__.py

```diff
@@ -1,26 +1,23 @@
 import sys
-import os
 import argparse
 import tempfile
 import string  # For generating a temp file for stdin
 import random  # For generating a temp file for stdin
-from .__init__ import __version__ as ipwbVersion
+from .__init__ import __version__ as ipwb_version
 
 # ipwb modules
 from . import replay
 from . import indexer
-from . import util as ipwbUtil
-
-from .util import IPWBREPLAY_HOST, IPWBREPLAY_PORT
+from . import util
 
 
 def main():
-    ipwbUtil.checkForUpdate()
-    args = checkArgs(sys.argv)
+    util.check_pypi_for_update()
```

Comment:
At one point long ago, we did call this variable `util` but ran into some naming clash. It might be moot with the move to Py3 and refinement of some extraneous dependencies. @ibnesayeed?

Reply:
Yes @machawk1 I recall you telling me about some conflicts long time ago when I asked why did you crate an alias for this import, but I think it should be alright now. Also, I think this import aliasing is done in other files too.